Makefile

```diff
@@ -3,28 +3,41 @@
 # Please refer to the LICENSE and AUTHORS files for details
 # SPDX-License-Identifier: MIT 
 
-CC := gcc
+CC     := gcc
+CFLAGS := -DPT_DEBUG -O2 -std=c99 -pedantic -Wall -Wextra
+
+LUA_DIR := /usr
+
 INSTALL_DIR    := /usr/local
 INSTALL_INCDIR := $(INSTALL_DIR)/include
 INSTALL_LIBDIR := $(INSTALL_DIR)/lib
+INSTALL_BINDIR := $(INSTALL_DIR)/bin
 
-.PHONY: install ptracer_header libptracer uninstall clean
+.PHONY: install ptracer_header pt-run libptracer uninstall clean
 
-install: ptracer_header libptracer
+install: ptracer_header pt-run libptracer
+	strip -s pt-run
```

Comment:
Why strip?
Let people have their debug symbols.

Reply:
Stripping binaries is a usual thing while install binaries. While building with `-O2` and no `-g` flag, there is no useful debugging symbol in the binary anyway. So, getting rid of the rest of the symbols should be just fine.

Reply:
I disagree about stripping being default practice. Please do not strip the binaries. Although some linux distributions do that, this is a decision best left for the downstream distributions, not the upstream project.